perfmetrics/scripts/fio/fio_metrics.py

```diff
@@ -363,6 +363,11 @@ def _extract_metrics(self, fio_out) -> List[Dict[str, Any]]:
 
     job_params = self._get_job_params(fio_out)
     start_end_times = self._get_start_end_times(fio_out, job_params)
+
+    # Print start and end time of jobs
+    for time in start_end_times :
```

Comment:
https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/perfmetrics/scripts/fetch_metrics.py#L39

please print start and endtimes here.

Reply:
fixed